src/witmotion_ros.cpp

```diff
@@ -89,6 +89,10 @@ ROSWitmotionSensorController::ROSWitmotionSensorController()
     : reader_thread(dynamic_cast<QObject *>(this))
 {
 
+  // In case we need string to float conversions this prevents locale dependant conversions
+  const char* locale = "C";
```

Comment:
Could we use here an overloaded constructor with the `std::string&` argument type, to drop away the C pointer type and lock for the newer C++ version?

Reply:
Of course, change it as you see best.  

I added this due to my problems with parameters being casted as strings. My machine's locale uses semicolon instead of dot when casting floats, so forcing "C" locale instead of mine ensured proper casting.

Reply:
@MFernandezCarmona could you please test the change and commit it, then I would merge?